docs/index.md

```diff
@@ -16,115 +22,237 @@
+PCA is almost entirely focused on retaining the global structure and variance of the data, and therefore one of its limitations is revealing patterns locally. 
+PCA is not an algorithm to be used on a similarity matrix, and is instead intended for transformed and normalized multidimensional data.
+In the context of this paper, PCA will be used on transformed and normalized genetic data and not on the similarity matrix described above.
+Because PCA retains global structure, few clear clusters different than the overall shape of the phylogenetic tree will be seen. 
 
-- Because PCA (Principal Component Analysis) reduces multidimensional data and not distance matrices, PCA was used to analyze the data in this format. While the ratio given by the within vs between violin plot was 24:1 and a positive R^2 correlation of .739, revealing a tightly clustered set of data, the data was not transformed to show any new pattern or information, and clustered the data almost identically to the .json rendering of the tree.
-<iframe src="https://blab.github.io/cartography/PCAFluBrush.html" style="width: 1200px; height: 400px;" frameBorder="0"></iframe>
 
-![](PCAViolinPlotFlu.png)
-![](PCAScatterplotLOESSFlu.png)
+MDS (Multidimensional Scaling) refers to statistical techniques that reduce the complexity of a data set by quantifying similarity judgments, increasing interpretability of local relational structures mired in the dataset.
+MDS outputs a “map” that spatially conveys the relationships among the items, with dissimilar data points farther away from each other (effectively clustering data based off of similarity). [@hout_papesh_goldinger_2012] 
+MDS requires a specific “placement” of each item to find the “true” solution wherein the distances correlate perfectly to the quantified similarity index (which is often impossible - MDS gets as close as possible to this).
+Some limitations to MDS include that only one symmetric matrix is allowed as input, and the scale of measurement is ordinal (non numerical).
+MDS preserves global patterns over local patterns, but the algorithm’s importance on translating dissimilarity to distance does preserve some larger patterns in local structure as well.
+In the context of this paper, MDS will cluster data into sparse “sections” of the map while not creating actual clusters. 
+The data’s true global shape will be more important to MDS than these clusters. 
 
-### Zika:
 
-- The Ratio given by the within vs between violin plot was 1:1 and a R^2 correlation of .011, revealing data not clustered at all. The chart reflects this, having no pattern whatsoever. 
+t-SNE (t-distributed Stochastic Neighbor Embedding) visualizes high-dimensional data by giving each datapoint a location in a 2 to 3 dimensional map.
+t-SNE is better than other techniques at creating a single map that reveals both local and global patterns in the dataset.
```

Comment:
Try to avoid quality judgements like this when describing methods. For our purposes, it's enough to state the benefits and limitations of t-SNE (and the other methods) in one or two sentences. Try condensing this paragraph to that length and repeat the process for the PCA, MDS, and U-MAP paragraphs. In the end, we should have one paragraph that quickly summarizes the benefits and limitations of each method. Readers can find technical details of the methods (e.g., "non-convexity of optimization stemming from the existence of multiple local minima") in the references you provide.